src/class-parsely.php

```diff
@@ -964,13 +964,31 @@ public function construct_parsely_metadata( array $parsely_options, $post ) {
 		/**
 		 * Filters the structured metadata.
 		 *
+		 * @deprecated deprecated since version 2.5.0
```

Comment:
```suggestion
		 * @deprecated 2.5.0 Use `wp_parsely_metadata` filter instead.
```

See https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#54-deprecated for recommended format of the `@deprecated` tag.